### ISSUE TITLE
Does it work on Windows?

### ISSUE THREAD

**Issue by georgeanderson** [patstockwell/vim-monokai-tasty#10]
I've tried using it in Windows with Neovim but it doesn't complete initializing nvim. Any tips on how to troubleshoot or gather more details about what's happening under the hood?

**Comment by georgeanderson** [comment]
Thanks @patstockwell. I won't have access to a Windows machine in the next few days until I am back to work, so expect some delay on my feedback there. Appreciate your quick response.

Merry Christmas!

**Comment by georgeanderson** [comment]
I did a fresh install and as soon as PlugInstall finished I could set it just fine and it worked. However, if I modify my init.vim (I'm using neovim) to include it by default it doesn't finish loading and dies with the following message:

```
λ nvim
Error detected while processing C:\Users\george_anderson\AppData\Local\nvim\init.vim:
line    9:
E185: Cannot find color scheme 'vim-monokai-tasty'
Press ENTER or type command to continue
```

I reverted ```colorscheme vim-monokai-tasty``` to my previous colorscheme and everything worked again. Any clues on why it is not initializing with neovim?
